repos/system_upgrade/common/libraries/tests/test_rhsm.py

```diff
@@ -1,29 +1,119 @@
 
+# External commands called by the RHSM library
+RHSM_LIST_CONSUMED = ('subscription-manager', 'list', '--consumed')
+RHSM_STATUS = ('subscription-manager', 'status')
+RHSM_RELEASE = ('subscription-manager', 'release')
+RHSM_LIST_ENABLED_REPOS = ('subscription-manager', 'repos', '--list-enabled')
+
+RHSM_STATUS_OUTPUT_NONSATELLITE = '''
++-------------------------------------------+
+   System Status Details
++-------------------------------------------+
+Overall Status: Current
+
+System Purpose Status: Matched
+'''
+
+RHSM_STATUS_OUTPUT_SATELLITE = '''
++-------------------------------------------+
+   System Status Details
++-------------------------------------------+
+Overall Status: Current
+
+System Purpose Status: Not Specified
+'''
+
+RHSM_STATUS_OUTPUT_SCA = '''
++-------------------------------------------+
+   System Status Details
++-------------------------------------------+
+Overall Status: Current
+
+System Purpose Status: Matched
+
+Content Access Mode is set to Simple Content Access
+'''
+
+# Used to simulate realistic output of RHSM, therefore carries more information than `Repository` namedtuple
+RHSMRepositoryEntry = namedtuple('RHSMRepositoryEntry', ('id', 'name', 'url', 'enabled'))  # For clarity purposes
+RHSM_ENABLED_REPOS = [
+    RHSMRepositoryEntry(
+        id='rhel-8-for-x86_64-appstream-rpms',
+        name='Appstream',
+        url='some_url',
+        enabled='1'),
+    RHSMRepositoryEntry(
+        id='satellite-tools-6.6-for-rhel-8-x86_64-rpms',
+        name='Satellite',
+        url='some_url',
+        enabled='1'),
+    RHSMRepositoryEntry(
+        id='rhel-8-for-x86_64-baseos-rpms',
+        name='Base',
+        url='some_url',
+        enabled='1')
+]
+
 
 class IsolatedActionsMocked(object):
     def __init__(self, call_stdout=None, raise_err=False):
         self.commands_called = []
         self.call_return = {'stdout': call_stdout, 'stderr': None}
         self.raise_err = raise_err
 
-    def call(self, cmd, *args):
+        # A map from called commands to their mocked output
+        self.mocked_command_call_outputs = dict()
+
+    def call(self, cmd, *args, **dummy_kwargs):
         self.commands_called.append(cmd)
         if self.raise_err:
             raise_call_error(cmd)
-        return self.call_return
+
+        return self.mocked_command_call_outputs.get(
+            tuple(cmd),  # Cast to tuple, as list is not hashable
+            self.call_return)
+
+    def add_mocked_command_call_with_stdout(self, cmd, stdout):
+        # We cast `cmd` from list to tuple, as a list cannot be hashed
+        self.mocked_command_call_outputs[tuple(cmd)] = {
+            'stdout': stdout,
+            'stderr': None}
+
+    def full_path(self, path):
+        return path
+
+
+@pytest.fixture
+def actor_mocked(monkeypatch):
+    """Fixture providing a mocked actor that was already used to monkeypatch api.current_actor.
+    Introduced to reduce repetition inside tests.
+    """
```

Comment:
```suggestion
    """
    Fixture providing a mocked actor that was already used to monkeypatch api.current_actor.
    
    Introduced to reduce repetition inside tests.
    """
```

We expect always one of these formats of docstrings
```
"""
title.

body
"""
```
or (without body)
```
"""
title.
"""
```
or
```
"""Title."""
```

Reply:
Understood, fixed and absorbed, so it should not be a problem in the future.